webpack/assets/javascripts/react_app/components/common/EmptyState/DefaultEmptyState.js

```diff
@@ -14,7 +14,7 @@ const documentationBlock = ({
   url && (
     <React.Fragment>
       {label}{' '}
-      <a href={url} target="_blank">
+      <a href={url} target="_blank" rel="noopener noreferrer">
```

Comment:
why?

Reply:
https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/